content/media/gigascience/convert-images.sh

```diff
@@ -0,0 +1,28 @@
+
+# Figure 1
+# TODO export content/media/rephetio/metagraph-and-features.psd to EPS via photo
+
+# Figure 2 and 3
+# see https://github.com/greenelab/connectivity-search-manuscript/issues/7
+# confirm that the source is not vector, hence an EPS version would not make sense
+# Figure 2 content/media/webapp/v3/b.metapaths-expanded.png
+# Figure 3 content/media/webapp/v3/webapp.png
```

Comment:
@vincerubinetti can you confirm that these figures are based on raster screenshots, such that there would be little benefit of an EPS version?

Reply:
Yes these are definitely raster, as they're just screenshots of the webapp. The individual sections seem to be high enough res that AI upscaling might work well here, if you need it.